eas.json

```diff
@@ -1,14 +1,19 @@
 {
   "cli": {
-    "version": ">= 0.53.0"
+    "version": ">= 0.53.0",
+    "appVersionSource": "remote"
   },
   "build": {
     "production": {
       "node": "18.19.1",
       "yarn": "1.22.19",
       "channel": "production",
       "ios": {
-        "image": "latest"
+        "image": "latest",
+        "autoIncrement": true
```

Comment:
does this auto increment reset to 1, when changing the main version for the app, as we always do manually?

Reply:
Very good question. I couldn't find anything about it in the document. I imagine it could increase continuously, but I can't answer that without testing it.

Reply:
ok, we will see. and i think it would be no problem for us to change to a continuously increasing version code, right?

Reply:
That's what I think. We're always going to increase it for Android. For iOS, there won't be any problem.

Reply:
I did some thinking yesterday. To be honest, I don't think the `autoIncrement` option would be right for us, because we need an extra commit to trigger the pipeline. If we have `autoIncrement`, we can't send an extra commit. I think we should continue as we have been doing so far and remove the `autoIncrement` option from the code. This way we can control and increment `versionCode` and `buildNumber` ourselves. This way we can also send an extra commit to trigger the pipeline.

Reply:
ok, we can have that in mind yes, when manually triggering per commit.